shared/utils.js

```diff
@@ -70,15 +70,16 @@ function setLogLevel(logLevel) {
  */
 function getVersionNumberFromString(versionString) {
 	// Only supporting major/minor version checking at this point.
-    var versionRegex = new RegExp(/^[^\d]*(\d+)(\.(\d+))?.*$/, 'm');
-	var matchArray = versionString.match(versionRegex);
+    var versionRegex = new RegExp(/^[^\d]*(\d+)(\.(\d+))(\.(\d+))?.*$/, 'm');
```

Comment:
Let me know if there is a better way to do this (my regex skills are poor).  This works for `1.7`, `1.7.2`, `1.7.2.2`, `1.7.2 junk`, etc.

Reply:
That looks right.